packages/babel-parser/test/fixtures/es2015/identifiers/invalid-escape-seq-export/output.json

```diff
@@ -0,0 +1,109 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 22,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 1,
+      "column": 22
+    }
+  },
+  "errors": [
+    "SyntaxError: Escape sequence in keyword export (1:15)",
+    "SyntaxError: Unexpected keyword 'export' (1:4)"
```

Comment:
This error should be removed.

Consider change test case to 
```js
expor\u{74} default 123;
```
options.json
```json
{
  "sourceType": "module"
}
```

Reply:
I think that it isn't what this test is for. Since it is in the `identifiers` folder, I think that here we are testing that `export` can't be used as an identifier, even if it contains escape sequences.

Reply:
The `identifiers/invalid-escape-seq-if` is actually testing invalid escape only:
```js
\u0069\u{66} (true) {}
```

We may create new test cases `invalid-escape-seq` alongside `identifiers` so we can test different error types.